keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -554,6 +555,8 @@ Upon success, the `BucketAccess` instance is ready and the app workload can acce
 + User creates a `BucketRequest` (BR) that references this `Bucket`.
 + User creates `BucketAccessRequest` (BAR) that references their BR, a `BucketAccessClass`, and their secret containing access tokens.
 + COSI detects the existence of the BAR and follows the [grant access](#grant-bucket-access) steps.
++ COSI detects the existence of the BAR, BA and executes a validation process.
++ Given a `BucketRequest` that the `BucketAccessRequest` is referencing, COSI checks to see that the `BucketAccess` referenced in the  `BucketAccessRequest` contains the same `Bucket` instance as the `Bucket` instance th the `BucketRequest` references.  
```

Comment:
I think I understand this step. Does this read any clearer to you?
```
+ As a validation step, COSI ensures that the `BucketRequest.bucketInstanceName` matches the `BucketAccessRequest.bucketAccessName.bucketInstanceName`. In other words, the two related `Bucket` references match, meaning that the BR and BAR->BA point to the same `Bucket`.
```

Reply:
I will make that change.